steering/src/tft.c

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
Right now the TFT code is duplicated. Working on a platformio library fix for this. But might be helpful for those working on steering to merge as is. Refactor in the next PR